### ISSUE TITLE
ffmpeg compile error

### ISSUE THREAD

**Issue by Weezzel** [m-ab-s/media-autobuild_suite#1731]
C:/MABS/build/ffmpeg-git/libavformat/libsrt.c: In function 'libsrt_set_options_pre':
C:/MABS/build/ffmpeg-git/libavformat/libsrt.c:317:66: error: 'SRTO_STRICTENC' undeclared (first use in this function); did you mean 'SRTO_STATE'?
  317 |         (s->enforced_encryption >= 0 && libsrt_setsockopt(h, fd, SRTO_STRICTENC, "SRTO_STRICTENC", &s->enforced_encryption, sizeof(s->enforced_encryption)) < 0) ||
      |                                                                  ^~~~~~~~~~~~~~
      |                                                                  SRTO_STATE
C:/MABS/build/ffmpeg-git/libavformat/libsrt.c:317:66: note: each undeclared identifier is reported only once for each function it appears in
C:/MABS/build/ffmpeg-git/libavformat/libsrt.c:336:50: error: 'SRTO_SMOOTHER' undeclared (first use in this function); did you mean 'SRTO_SENDER'?
  336 |         (s->smoother && libsrt_setsockopt(h, fd, SRTO_SMOOTHER, "SRTO_SMOOTHER", s->smoother, strlen(s->smoother)) < 0) ||
      |                                                  ^~~~~~~~~~~~~
      |                                                  SRTO_SENDER
C:/MABS/build/ffmpeg-git/libavformat/libsrt.c: In function 'libsrt_setup':
C:/MABS/build/ffmpeg-git/libavformat/libsrt.c:409:5: warning: 'srt_socket' is deprecated [-Wdeprecated-declarations]
  409 |     fd = srt_socket(cur_ai->ai_family, cur_ai->ai_socktype, 0);
      |     ^~
In file included from C:/MABS/build/ffmpeg-git/libavformat/libsrt.c:24:
C:/MABS/local64/include/srt/srt.h:735:41: note: declared here
  735 | SRT_ATR_DEPRECATED_PX SRT_API SRTSOCKET srt_socket(int, int, int) SRT_ATR_DEPRECATED;
      |                                         ^~~~~~~~~~
make: *** [/build/ffmpeg-git/ffbuild/common.mak:59: libavformat/libsrt.o] Error 1

[logs.zip](https://github.com/m-ab-s/media-autobuild_suite/files/4864144/logs.zip)

**Comment by 1480c1** [comment]
Duplicate of #1725
